### DISCUSSION TITLE
Is it possible to get class attribute type docstrings in the same line as the attribute name?

### DISCUSSION THREAD

**Discussion by jtpavlock** [pradyunsg/furo#533] [Questions] [answered]
For class attributes, instead of:
![image](https://user-images.githubusercontent.com/26807251/192166478-5b9df40b-ffca-4e7d-ada9-9f34ab66ff9f.png)

I'd like the type to be in the same line, such as when properties get documented:
![image](https://user-images.githubusercontent.com/26807251/192166527-dbccdaa7-4916-4e89-8921-21d573df5755.png)

**Comment by jtpavlock** [comment]
Can this be changed to a feature request if it is not currently possible?

**Comment by pradyunsg** [comment, marked as answer]
I don't believe so -- this would have to be a Sphinx feature request, since Furo is merely stylising the markup it gets from Sphinx (Furo can only affect the actual page *content* via CSS, the HTML in it is for all the scaffolding around it).

**Reply by jtpavlock** [reply]
Made a [feature request](https://github.com/sphinx-doc/sphinx/issues/10909) with sphinx, thanks.
